lib/Controller/WopiController.php

```diff
@@ -223,6 +223,7 @@ public function checkFileInfo($fileId, $access_token) {
 			'DownloadAsPostMessage' => $wopi->getDirect(),
 			'SupportsLocks' => $this->lockManager->isLockProviderAvailable(),
 			'IsUserLocked' => $this->permissionManager->userIsFeatureLocked($wopi->getEditorUid()),
+			'EnableRemoteLinkPicker' => (bool)$wopi->getCanwrite(),
```

Comment:
I think the link picker should for now also be disabled for public links and direct editing (from the mobile clients)

Reply:
True, not sure about the new conditions.